website/content/partials/known-issues/1_15-audit-file-sighup-does-not-trigger-reload.mdx

```diff
@@ -0,0 +1,31 @@
+### File audit devices do not honor SIGHUP signal to reload
+
+#### Affected versions
+
+- 1.15.0
+
+#### Issue
+
+Sending [`SIGHUP`](/vault/docs/audit/file#log-file-rotation) to Vault after rotating
+an audit log file doesn't reopen the file path as expected.
```

Comment:
```suggestion
The new underlying event framework for Vault causes Vault to continue using 
audit logs instead of reopening the file paths even when you send 
[`SIGHUP`](/vault/docs/audit/file#log-file-rotation) after log rotation:
```

Style correction: write in active voice, make it clear who/what takes action, avoid contractions